### PR TITLE
Generalization on learn/framework/modules/

### PR DESCRIPTION
**Pull request by augustocollado**
### Description

This PR generalizes modules sections:
It renames pallets to modules, and also removes some irrelevenat info about system and parachains pallets.

### Checklist

- [ ] I have added a label to this PR 🏷️
- [ ] I have run my changes through Grammarly
- [ ] If this page requires a disclaimer, I have added one
- [ ] If pages have been moved around, I have created an additional PR in `tanssi-mkdocs` to update redirects

### CODE REVIEW

File: learn/framework/modules.md [hunk 1]
```diff
@@ -9,34 +9,31 @@ description: Substrate is a blockchain development framework that provides modul
...
 
-When it comes to building the Runtime, which is essentially the heart of a Substrate-based blockchain, the desired state transition rules must be defined, reflecting the intended behavior and features of the blockchain.
+When it comes to building the Runtime, which is essentially the heart of a Tanssi appchain, the desired state transition rules must be defined, reflecting the intended behavior and features of the blockchain.
 
-To build the Runtime, Substrate provides many built-in modules (called pallets) that can be freely used as building blocks to compose and interact with any other custom-made modules, allowing teams to create unique behaviors according to the specific requirements of their Tanssi appchain.
+To build the Runtime, Substrate provides many built-in modules (AKA pallets) that can be freely used as building blocks to compose and interact with any other custom-made modules, allowing teams to create unique behaviors according to the specific requirements of their Tanssi appchain.
```
Comment (albertov19) [inline learn/framework/modules.md:14]:
```suggestion
To build the Runtime, Substrate provides many built-in modules (also known as pallets) that can be freely used as building blocks to compose and interact with any other custom-made modules, allowing teams to create unique behaviors according to the specific requirements of their Tanssi appchain.
```

File: learn/framework/modules.md [hunk 2]
```diff
@@ -9,34 +9,31 @@ description: Substrate is a blockchain development framework that provides modul
...
-- **[pallet_multisig](https://paritytech.github.io/substrate/master/pallet_multisig/index.html){target=\_blank}** - the Multisig pallet provides functions for multi-signature dispatch
-- **[pallet_recovery](https://paritytech.github.io/substrate/master/pallet_recovery/index.html){target=\_blank}** - the Recovery pallet provides functions to allow users to regain access to their accounts when the private key is lost. This works by granting other accounts the right to sign transactions on behalf of the lost account (note that it is necessary to have previously chosen the authorized accounts)
-- **[pallet_staking](https://paritytech.github.io/substrate/master/pallet_staking/index.html){target=\_blank}** - the Staking pallet provides functions to administer staked tokens, support rewarding, slashing, depositing, withdrawing, and so on
+- **[pallet_balances](https://paritytech.github.io/substrate/master/pallet_balances/index.html){target=\_blank}** - the Balances modules provides functions for handling accounts and balances for the Tanssi appchain native currency
```
Comment (albertov19) [inline learn/framework/modules.md:24]:
```suggestion
- **[Balances module](https://paritytech.github.io/substrate/master/pallet_balances/index.html){target=\_blank}** - it provides functions for handling accounts and balances for the Tanssi appchain native currency
```

Comment (albertov19) [reply learn/framework/modules.md:24]: I suggest we do the others this way :)

**Review by albertov19 — CHANGES_REQUESTED** [review]
Some more tiny nitpicks

**Review by albertov19 — CHANGES_REQUESTED** [review]
Some notes

File: learn/framework/modules.md [hunk 3]
```diff
@@ -9,34 +9,31 @@ description: Substrate is a blockchain development framework that provides modul
...
-- **[pallet_multisig](https://paritytech.github.io/substrate/master/pallet_multisig/index.html){target=\_blank}** - the Multisig pallet provides functions for multi-signature dispatch
-- **[pallet_recovery](https://paritytech.github.io/substrate/master/pallet_recovery/index.html){target=\_blank}** - the Recovery pallet provides functions to allow users to regain access to their accounts when the private key is lost. This works by granting other accounts the right to sign transactions on behalf of the lost account (note that it is necessary to have previously chosen the authorized accounts)
-- **[pallet_staking](https://paritytech.github.io/substrate/master/pallet_staking/index.html){target=\_blank}** - the Staking pallet provides functions to administer staked tokens, support rewarding, slashing, depositing, withdrawing, and so on
+- **[Balances module](https://paritytech.github.io/substrate/master/pallet_balances/index.html){target=\_blank}** - it provides functions for handling accounts and balances for the Tanssi appchain native currency
```
[the remark this answers is not in the document]
Comment (augustocollado) [reply learn/framework/modules.md:24]: Ah shoot! yes, will do

[the remark this answers is not in the document]
Comment (augustocollado) [reply learn/framework/modules.md:24]: Wait. that was in this same PR, and it's already done -> https://github.com/moondance-labs/tanssi-docs/pull/195/commits/414c23300fe5c3bddf8b7a031908e97eb8b4308c

[the remark this answers is not in the document]
Comment (albertov19) [reply learn/framework/modules.md:24]: Nah but I mean the actual word module that is repeated multiple times in each bullet point

**Review by albertov19 — APPROVED** [review]
LGTM!

**Pull request merged**